code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -49,7 +49,14 @@
 
 /datum/reagent/slimejelly/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
-	if(M.get_blood_id() != id)  // no effect on slime people
+	var/mob/living/carbon/C = M
+	if(iscarbon(C) && C.mind?.has_antag_datum(/datum/antagonist/vampire))
+		M.set_nutrition(min(NUTRITION_LEVEL_WELL_FED, M.nutrition + 10))
```

Comment:
Why not adjust nutrition? This seems like it'd instantly bring people up to well fed with just one sip.

Reply:
As explained, both of these are (Sadly) how they already work, I copy pasted em from how normal blood works. Either me or someone else can make a new PR later on for this